components/filterTodos.js

```diff
@@ -0,0 +1,35 @@
+import { renderTodoList } from "./domActions.js";
+import { setCurrentFilterState } from "../entry.js";
+import {
+  filterButtonWrapper,
+  filterAllButton,
+  filterCompleteButton,
+  filterIncompleteButton,
+} from "./domElements.js";
+
+const filterButtons = filterButtonWrapper.getElementsByClassName("button");
+
+function selectedClass(clicked) {
+  for (let current of filterButtons) {
+    current.classList.remove("active");
+  }
+  clicked.classList.add("active");
+}
+
+export function filterAll() {
+  setCurrentFilterState("all");
```

Comment:
Let's use const here. 
```suggestion
  import { ALL, COMPLETE, INCOMPLETE } from "../const.js"
  setCurrentFilterState(ALL);
```